x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/utils/ingest/PipelineRegistry.java

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
ℹ️ This class may be moved to xpack core but I preferred to keep the code in our utils for now.

Reply:
From my perspective, it makes sense to move it to `x-pack/core` from the beginning.

Reply:
I would prefer to move it later because it will require other teams to validate which can add an additional delay to gets reviewed.

Reply:
@kderusso Being very close to FF is indeed part of my rationale here.